assets/templates/datasetLandingPage/filterable.tmpl

```diff
@@ -54,7 +54,7 @@
                 <ul class="list--neutral">
                 {{range $i, $download := $v.Downloads}}
                 {{if gt (len $download.Size) 0}}{{if gt (len $download.Size) 0}}<li class="padding-left--1 margin-top--0 margin-bottom--1 white-background clearfix"><span class="inline-block padding-top--2">{{if eq $download.Extension "txt"}}Supporting information{{else}}Complete dataset (<span class="uppercase">{{$download.Extension}}</span> format){{end}}</span>
-                  <div class="width--12 inline-block float-right text-right"><a id="{{$download.Extension}}-download"                                     data-gtm-download-file="{{$download.URI}}"
+                  <div class="width--12 inline-block float-right text-right float-left-mobile"><a id="{{$download.Extension}}-download"                                     data-gtm-download-file="{{$download.URI}}"
```

Comment:
I think we have some classes in the utilities file in Sixteens that I think should allow you to float elements and align text - `float-el--{left/right}` and `.text-{left/right}` - could they work instead perhaps? If so, we wouldn't need the new class (and save you a Sixteens PR in the process)

Reply:
Done! @rav-pradhan Thanks.